vispy/visuals/filters/mesh.py

```diff
@@ -181,9 +180,7 @@ def _attach(self, visual):
         // Note(asnt): The normal calculated above always points in the
         // direction of the camera. Reintroduce the original orientation of the
         // face.
-        if (!gl_FrontFacing) {
-            normal = -normal;
-        }
+        normal = gl_FrontFacing ? normal : -normal;
```

Comment:
Just curious, does this perform differently in the shader or is this change purely style?

Reply:
You want to avoid branches in shaders, and a ternary operation is more trivial to convert into branch-free code by the compiler.